data/json/items/armor/head_attachments.json

```diff
@@ -80,7 +80,7 @@
     "armor": [ { "coverage": 60, "covers": [ "mouth" ], "specifically_covers": [ "mouth_chin" ] } ]
   },
   {
-    "id": "counterweight pouch",
+    "id": "counterweight_pouch",
```

Comment:
That thing never spawned in game so far? Because otherwise changing the id will break saves

Reply:
As far as a know, yes. I checked if it spawns anywhere on The Hitchhiker's Guide to the Cataclysm and it says that it doesn't spawn at all. [Link](https://nornagon.github.io/cdda-guide/#/item/counterweight%20pouch)
And for comparison [Link2](https://nornagon.github.io/cdda-guide/#/item/attachable_ear_muffs)